library/src/main/java/com/chuckerteam/chucker/internal/support/TeeSource.kt

```diff
@@ -18,12 +18,12 @@ import java.io.IOException
  */
 internal class TeeSource(
     private val upstream: Source,
-    private val sideChannel: File,
+    private val sideChannel: File?,
     private val callback: Callback,
     private val readBytesLimit: Long = Long.MAX_VALUE
 ) : Source {
     private val sideStream = try {
-        Okio.buffer(Okio.sink(sideChannel))
+        sideChannel?.let { Okio.buffer(Okio.sink(it)) }
```

Comment:
Same suggestion about using `if (sideChannel != null)` as in the previous comment.